apps/condo/domains/user/access/RegisterUserExternalIdentityService.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Generated by `createservice user.RegisterUserExternalIdentityService`
+ */
+const access = require('@open-condo/keystone/access')
+
+async function canRegisterUserExternalIdentity (args) {
```

Comment:
It looks like we don't need it. It's better to just create it as server-side util

Reply:
removed